native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -1257,7 +1351,10 @@ exports[`Android BpkButton should support the "iconOnly" and "large" property 1`
               "paddingLeft": 16,
               "paddingRight": 16,
             },
-            undefined,
+            Object {
```

Comment:
what's this bit here? from undefined to an object?

Reply:
I'm not 100% but I think it's some styling that wasn't rendered properly before and came out like that, which is working now.